src/components/CreateResolution/CompleteResolution.vue

```diff
@@ -0,0 +1,827 @@
+
+// Mixins
+import { CommonMixin, DateMixin, EntityFilterMixin } from '@/mixins'
+
+// Validation
+import { Rules } from '@/rules'
+
+@Component({
+  components: {
+    DatePicker
+  }
+})
+export default class CompleteResolution extends Mixins(CommonMixin, DateMixin, EntityFilterMixin) {
+  // Refs
+  $refs!: {
+    resolutionDatePickerRef: DatePicker,
+    resolutionTextFormRef: FormIF,
+    resolutionTextRef: FormIF,
+    signatureDatePickerRef: DatePicker,
+    signingPersonFormRef: FormIF,
+    signingPersonGivenNameRef: FormIF,
+    signingPersonMiddleNameRef: FormIF,
+    signingPersonFamilyNameRef: FormIF,
+    confirmResolutionChkFormRef: FormIF,
+    confirmResolutionChkRef: FormIF
+  }
+
+  private MAX_RESOLUTION_TEXT_LENGTH: number = 1000
+  private resolutionText: string = ''
+  private hasResolutionConfirmed = false
+  private resolutionConfirmed = false
+  private helpToggle = false
+  private PLACEHOLDER_LEGAL_NAME = 'legal_name'
+  private PLACEHOLDER_LEGAL_NAME_INLINE = 'legal_name_inline'
+  private signingPerson: PersonIF = null
+
+  // Date properties
+  private resolutionDateText: string = ''
+  private signatureDateText: string = ''
+  private foundingDate: Date = null
+
+  @Getter getShowErrors!: boolean
+  @Getter isTypeCoop!: boolean
+  @Getter getNameRequestDetails!: NameRequestDetailsIF
+  @Getter getCreateResolutionResource!: CreateResolutionResourceIF
+  @Getter getCreateResolutionStep!: CreateResolutionIF
+  @Getter getValidateSteps!: boolean
+  @Getter getBusinessLegalName!: string
+  @Getter getBusinessFoundingDate!: string
+
+  @Action setResolution!: ActionBindingIF
+  @Action setResolutionStepValidationDetail!: ActionBindingIF
+
+  // Enum for template
+  readonly RouteNames = RouteNames
+  readonly ItemTypes = ItemTypes
+  readonly CorpTypeCd = CorpTypeCd
+  readonly BulletListTypes = BulletListTypes
+  readonly Rules = Rules
+
+  private get documentURL (): string {
+    const docUrl = sessionStorage.getItem('BASE_URL') +
+      this.getCreateResolutionResource.sampleFormSection.downloadDocPath
+    return docUrl
+  }
+
+  private previewImageSource (): string {
+    // Note: the image file path did not resolve correctly when using the require function directly.  In order
+    // to get the image path resolving correctly, needed to get the image context first and use that to build
+    // the final image file path
+    const images = require.context('@/assets/images/', false, /\.png$/)
+    return images('./' + this.getCreateResolutionResource.sampleFormSection.previewImagePath)
+  }
+
+  private get confirmLabel (): string {
+    const confirmTextItems = this.getCreateResolutionResource.confirmSection.text
+    const result = confirmTextItems.map(item => {
+      if (item.type === ItemTypes.PLACEHOLDER && item.value === this.PLACEHOLDER_LEGAL_NAME) {
+        return `<strong>${this.getBusinessLegalName}</strong>`
+      }
+      return item.value
+    }).join(' ')
+    return result
+  }
```

Comment:
Seems like a lot for a dynamic label, are the `value`'s of the `confirmTextItems` much different than the html template returned there?
Is the main purpose to provide the `strong` tag in the event the conditions are met?

Reply:
the purpose of this is not just the strong tag but to allow swapping of the placeholder value as well.